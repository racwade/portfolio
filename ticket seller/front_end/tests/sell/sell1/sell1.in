login
test_sell
sell

logout