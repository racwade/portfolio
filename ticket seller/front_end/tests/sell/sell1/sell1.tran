00 test_sell           SS 000009999