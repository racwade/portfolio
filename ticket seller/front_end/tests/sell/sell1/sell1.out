welcome to the ticket seller system
enter username:
logged in as test_sell
enter event name:
error: must specify a valid event name
bye